cmd/bootstrap/cmd/access_keygen.go

```diff
@@ -0,0 +1,145 @@
+package cmd
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/model/bootstrap"
+	"github.com/onflow/flow-go/utils/grpcutils"
+	"github.com/onflow/flow-go/utils/io"
+)
+
+const certValidityPeriod = 100 * 365 * 24 * time.Hour // ~100 years
+
+var (
+	flagCommonName     string
+	flagNodeInfoFile   string
+	flagOutputKeyFile  string
+	flagOutputCertFile string
+)
+
+var accessKeyCmd = &cobra.Command{
+	Use:   "access-keygen",
+	Short: "Generate access node grpc TLS key and certificate",
+	Run:   accessKeyCmdRun,
+}
+
+func init() {
+	rootCmd.AddCommand(accessKeyCmd)
+
+	accessKeyCmd.Flags().StringVar(&flagNodeInfoFile, "node-info", "", "path to node's node-info.priv.json file")
+	_ = accessKeyCmd.MarkFlagRequired("node-info")
+
+	accessKeyCmd.Flags().StringVar(&flagOutputKeyFile, "key", "./access-tls.key", "path to output private key file")
+	accessKeyCmd.Flags().StringVar(&flagOutputCertFile, "cert", "./access-tls.crt", "path to output certificate file")
+	accessKeyCmd.Flags().StringVar(&flagCommonName, "cn", "", "common name to include in the certificate")
```

Comment:
fwiw: Common Name was deprecated by [rfc2818](https://datatracker.ietf.org/doc/html/rfc2818#section-3.1) (May 2000).

Reply:
added support for SANs c6614d6